lib/common/schemas.c

```diff
@@ -1331,3 +1331,115 @@ pcmk__schemas_later_than(const char *name)
 
     return lst;
 }
+
+static void
+external_refs_in_schema(GList **list, const char *contents)
+{
+    xmlNode *ptr = string2xml(contents);
+    /* local-name()= is needed to ignore the xmlns= setting at the top of
+     * the XML file.  Otherwise, the xpath query will always return nothing.
+     */
+    xmlXPathObjectPtr xpathObj = xpath_search(ptr, "//*[local-name()='externalRef'] | //*[local-name()='include']");
+
+    for (int i = 0; i < numXpathResults(xpathObj); i++) {
+        xmlNode *result = getXpathResult(xpathObj, i);
+        const char *href = NULL;
+
+        if (result == NULL) {
+            continue;
+        }
+
+        href = crm_element_value(result, "href");
+
+        if (href != NULL) {
+            char *s = strdup(href);
+
+            CRM_ASSERT(s != NULL);
+            *list = g_list_append(*list, s);
```

Comment:
Incidentally, I know we typically like to use g_list_prepend and then reverse.  There aren't a ton of external refs in any schema, but it's not nothing.  On the other hand, we're not exactly committed to using g_list_prepend...
```
clumens@spire ~/src/pacemaker schemas> grep -r g_list_prepend | wc -l 
91
clumens@spire ~/src/pacemaker schemas> grep -r g_list_append | wc -l
136
```

But maybe it's not worth making that problem worse.